core/sys/orca/orca.odin

```diff
@@ -0,0 +1,2183 @@
+//+build orca
```

Comment:
```suggestion

```
That should cause docs to be generated too, if you alternatively want to keep the tag it should also be added to the macros file